apps/wix/README.md

```diff
@@ -0,0 +1,26 @@
+# Wix Integration app
+
+The app can be installed using this [link](https://app.contentful.com/deeplink?link=apps&id=5oytqBOh7xKU40F5c28zQE)
```

Comment:
Can you remove this line? We will point people to the marketplace instead to install the app, and your app will have a new id.

Reply:
Done